sygnal/apnspushkin.py

```diff
@@ -230,6 +230,7 @@ async def _dispatch_request(
             message=shaved_payload,
             priority=prio,
             notification_id=notif_id,
+            push_type=PushType.ALERT,
```

Comment:
The apple docs imply that if we use the alert push type, we have to use the non-voip topic:
https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns/

> **alert**
> Use the alert push type for notifications that trigger a user interaction—for example, an alert, badge, or sound. If you set this push type, the apns-topic header field must use your app’s bundle ID as the topic. For more information, see Generating a remote notification.
>
> **voip**
> [...] If you set this push type, the apns-topic header field must use your app’s bundle ID with .voip appended to the end.

However it seems that we configure sygnal to use both voip and non-voip topics:
```
    - name: im.vector.app.ios.prod
      [...]
      topic: im.vector.app

    - name: im.vector.app.ios.dev
      platform: sandbox
      [...]
      topic: im.vector.app

    - name: im.vector.app.ios.voip.prod
      [...]
      topic: im.vector.app.voip

    - name: im.vector.app.ios.voip.dev
      platform: sandbox
      [...]
      topic: im.vector.app.voip
```

I think it's best that we make a push_type config option that defaults to `None`.
We could then roll it out to dev builds first, before turning it on for production builds.